code/modules/mob/living/simple_animal/friendly/snake.dm

```diff
@@ -45,17 +45,25 @@
 			mice += HM
 		if(isliving(HM))
 			living_mobs += HM
-
 	// if no tasty mice to chase, lets chase any living mob enemies in our vision range
 	if(length(mice) == 0)
 		//Filter living mobs (in range mobs) by those we consider enemies (retaliate behaviour)
 		return  living_mobs & enemies
 	return mice
 
 /mob/living/simple_animal/hostile/retaliate/poison/snake/AttackingTarget()
-        if(istype(target, /mob/living/simple_animal/mouse))
-                visible_message("<span class='notice'>[name] consumes [target] in a single gulp!</span>", "<span class='notice'>You consume [target] in a single gulp!</span>")
-                QDEL_NULL(target)
-                adjustBruteLoss(-2)
-        else
-                return ..()
\ No newline at end of file
+	if(istype(target, /mob/living/simple_animal/mouse))
+		visible_message("<span class='notice'>[name] consumes [target] in a single gulp!</span>", "<span class='notice'>You consume [target] in a single gulp!</span>")
```

Comment:
And to think, there was a problem with me making plushies get mashed together for the sole purpose of procreation.

But now it is the frog who is introducing vore to our fair tgstation13.

The slipper is on the other head now!

Reply:
But we already had vore with Alien Hunters.

Reply:
Our vore code's beginning was xenomorph devour_mob